easybuild/easyconfigs/c/CRISPR-DAV/CRISPR-DAV-2.3.4-foss-2020b.eb

```diff
@@ -0,0 +1,36 @@
+name = 'CRISPR-DAV'
+version = '2.3.4'
+
+homepage = 'https://github.com/pinetree1/crispr-dav/'
+description = """CRISPR-DAV is a pipeline to analyze
+amplicon-based NGS data of CRISPR clones in a high throughput manner."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/pinetree1/crispr-dav/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['crispr-dav-2.3.4_remove_hardcoding.patch']
```

Comment:
please rename the patch file to `CRISPR-DAV-2.3.4_remove_hardcoding.patch`, and use `%(version)s` here:

```python
patches = ['CRISPR-DAV-%(version)s_remove_hardcoding.patch']